plugins/tiledb/io/Bounds4D.cpp

```diff
@@ -0,0 +1,406 @@
+//
```

Comment:
This is auto-created from clion but can we change this to match the TileDB header in TileDBWriter.cpp? Your work will still be reflected in the commit.